### PR TITLE
Fix schema issues

### LINKED ISSUE #24: Error when pulling ems-incident-dispatch
```shell
kamu pull us.cityofnewyork.data.ems-incident-dispatch 
  [1/7] Checking for updates (us.cityofnewyork.data.ems-incident-dispatch)                                                                                                                                 
  Downloading us.cityofnewyork.data.ems-incident-dispatch:
  [00:31:31] [#########################################################################################################################################################] 5.88 GiB/5.88 GiB (3.18 MiB/s, 0s)  [4/7] Failed to update root dataset (us.cityofnewyork.data.ems-incident-dispatch)                                                                                                                        Error: Failed to update 1 dataset(s)

Summary of individual errors:

> Failed to pull us.cityofnewyork.data.ems-incident-dispatch:
  0: Internal error
  1: This feature is not implemented: Unsupported SQL type Custom(ObjectName([Ident { value: "LONG", quote_style: None }]), [])
```

### PR DESCRIPTION
**Pull request by andriidemus**
Closes: #24

### CODE REVIEW

**Comment by andriidemus** [comment]
Original data set is up to 6GB of size. Here's cut cache file for testing
[Archive.zip](https://github.com/user-attachments/files/17721496/Archive.zip)

File: us.cityofnewyork.data/ems-incident-dispatch/daylight_savings_effect.ipynb [hunk 1]
```diff
@@ -26,7 +26,39 @@
    "source": [
     "%%sql\n",
     "CREATE OR REPLACE TEMPORARY VIEW dispatch_serious AS (\n",
-    "    SELECT * FROM dispatch\n",
+    "    SELECT\n",
```
Comment (sergiimk) [inline us.cityofnewyork.data/ems-incident-dispatch/daylight_savings_effect.ipynb:29]: Was `select *` somehow problematic?

File: us.cityofnewyork.data/ems-incident-dispatch/ems-incident-dispatch.yaml [hunk 2]
```diff
@@ -11,39 +11,75 @@ content:
...
+          - policeprecinct BIGINT  # The police precinct of the incident.
+          - citycouncildistrict BIGINT  # The city council district.
+          - communitydistrict BIGINT  # The community district.
+          - communityschooldistrict BIGINT  # The community school district.
+          - congressionaldistrict BIGINT  # The congressional district.
           - reopen_indicator STRING  # Indicates that at some point the incident was closed but then reopened.
           - special_event_indicator STRING  # Indicates that the incident was a special event such as the NYC Marathon.
           - standby_indicator STRING  # Indicates that the units were assigned to stand by in case they were needed.
           - transfer_indicator STRING  # Indicates that the incident was created for the transportation of a patient from one facility (ie a hospital or nursing home) to another.
+      preprocess:
```
Comment (sergiimk) [inline us.cityofnewyork.data/ems-incident-dispatch/ems-incident-dispatch.yaml:47]: Can you please add a `# TODO: ` comment above pointing at the issue to support custom `timestampFormat` in DataFusion ingest?

**Comment by sergiimk** [comment]
Please convert to a full PR once this ready to be merged and I can give it a quick test. It's ok if you didn't get all the notebooks working - `kamu-contrib` repo is treated as "best effort" support, so things often go a bit outdated. We'll eventually get to it if this dataset proves useful.

File: us.cityofnewyork.data/ems-incident-dispatch/daylight_savings_effect.ipynb (hunk 1 above)
Comment (andriidemus) [reply us.cityofnewyork.data/ems-incident-dispatch/daylight_savings_effect.ipynb:29]: Yes, because of `offset` column presence.

**Review by sergiimk — APPROVED** [review]

**Pull request merged**